### DISCUSSION TITLE
Mi Band WatchFace Editor GUI

### DISCUSSION THREAD

**Discussion by Johnson070** [Johnson070/MiBand-5-watchface-editor#28] [General]
1.0.1.7 (2021-05-26)

- Add see Global chinese and Simple chinese watchface

- Bug fixes in date form

- Start work for Simple chinese language

Thank you for downloading the program.

If you find a bug, please write to github or telegram: @vveber

<hr /><em>This discussion was created from the release <a href='https://github.com/Johnson070/MiBand-5-watchface-editor/releases/tag/1.0.1.7'>Mi Band WatchFace Editor GUI</a>.</em>

**Comment by Mike12358-ai** [comment]
Hey, buddy. One problem I found is that with your dial maker, if you change it Distance There will be blank data such as heart rate and steps on the bracelet 
There is no problem with previewing 
As long as you change the distance between the distances, there will be problems with the dial. The bracelet will not display the status, the number of steps, the heart rate distance, and the consumption will disappear

**Comment by Mike12358-ai** [comment]
麻烦修复一下，更改“距离”数字的间距，就会造成手环，心率，步数一切状态在手环上空白

**Comment by Mike12358-ai** [comment]
Trouble to fix it, change the spacing of the "distance" numbers, it will cause the bracelet, heart rate, steps all the state on the bracelet blank

**Reply by Johnson070** [reply]
Ok, I will see

**Reply by Mike12358-ai** [reply]
Did you find anything wrong

**Comment by 104bertchen** [comment]
1. A bug in Weather.Wind child element, editor uses "ImageSufficCN2" instead of "ImageSuffixCN2" and causes the bin packer failed to pack.
2. I have also figured out that when add Weather.Wind, as well as Weather.AirPollution, all weather info will be blank when upload to mi band, if you remove it, everything works fine.
3. Bluetooth ON/OFF switch setting is inverted, when add ON image it's actually used as OFF in mi band and vice versa.
4. The element Date.ENWeekDays (CNWeekDays, CN2WeekDays also) does not work in mi band, instead I use "WeekDaysIcons" element and works fine
5. The preview renderer incorrectly interprets StartAngle and EndAngle in CircleScale of  StepsProgress (Calories & Heart as well). However, animated GIF which is generated by bin packer has correct interpretation. This will get people confused.

**Reply by Johnson070** [reply]
1. Find, thanks
2. Try change language
3. Fix in update
4. i don't know
5. Thanks, i find bug

**Reply by Johnson070** [reply]
Thank you so much for your help
